### PR TITLE
1: add file path adding, and form data format

### PR DESCRIPTION
**Pull request by Rohlinsky**

### CODE REVIEW

**Comment by kennym** [comment]
Can you describe what this PR does?

**Comment by Rohlinsky** [comment]
Sure, 
194 - 199: Just comfort view of attributes input
456 - 462: Go over the fields to find witch is file, not text, and mark it with '@', before value.
577, 637: Auto fix of beautifier.
651 - 655: Check request type, if this is default type 'params', just send it throw 'form-data', other (json, xml) send default 'wp_remote_post' method.
685 - 734: Function which provide possibility to send file and other types of fields through 'form-data', with link of author above, little bit modified to comfort catch all files in inputs, and append it to request body.

**Comment by kennym** [comment]
looking good! Once that variable is fixed I'll merge this

**Comment by Rohlinsky** [comment]
Can you please mark, witch variable I should fix, and I fix it.

File: includes/class-cf7-api-admin.php [hunk 1]
```diff
@@ -448,6 +453,13 @@ function clear_error_log( $post_id ){
   function get_record( $submission , $qs_cf7_data_map , $type = "params", $template = "" ){
 
     $submited_data = $submission->get_posted_data();
+    if ($type == "params") {
+      $files = $submission->uploaded_files();
+      foreach ($files as $key => $value) {
+        $submited_data[$key] = '@"'.$value.'"';
```
[the remark this answers is not in the document]
Comment (kennym) [reply includes/class-cf7-api-admin.php:459]: Sorry, I never submitted this review.

**Pull request closed without merging**